docs/tutorial/part-four/index.md

```diff
@@ -211,7 +211,7 @@ error-prone, especially as sites get larger and more complex. It's much better t
 store the title in one place and then _pull_ that title into components whenever
 we need it.
 
-To solve this, Gatsby supports a simple pattern for adding site "metadata"—like
+To solve this, Gatsby supports the following pattern for adding site "metadata"—like
```

Comment:
Similar to above, could probably say “to solve this we can add site metadata—like the title— to the Gatsby config file.” And then go on with the example.